### ISSUE TITLE
Files and streams

### ISSUE THREAD

**Issue by synw** [jozews/nearby-connectivity#1]
Hi, nice plugin. Reading the Nearby connectivity doc it says that you can transfer files and also use streams. The readme page only mentions bytes transfer, and the doc says bytes payloads are limited to 32k. It would be great to be able to transfer files.

A complete example would help: I have seen the folder but I guess it is unfinished yet.

**Comment by jozews** [comment]
Thats correct. Currently only bytes are supported as this was the specific use case I had.
Feel free to implement a new sendPayload method and extend the Payload model class or add new PayloadFile and PayloadStream classes.

**Comment by mannprerak2** [comment]
is the byte payload working? I can't really see any implementation on android side or even iOS code
